lib/committee/rails/test/methods.rb

```diff
@@ -3,24 +3,51 @@ module Test
     module Methods
       include Committee::Test::Methods
 
-      def schema_path
-        Rails.root.join('docs', 'schema', 'schema.json')
+      def committee_schema
+        @committee_schema ||= begin
+          driver = Committee::Drivers::HyperSchema.new
+          schema_hash = JSON.parse(File.read(Rails.root.join('docs', 'schema', 'schema.json')))
```

Comment:
now spec override schema_path method likes following. so it's green 💚 .

[committee-rails/methods_spec.rb at 05a85120e5504eb0a9ebbd617874270541b6be01 · willnet/committee-rails](https://github.com/willnet/committee-rails/blob/05a85120e5504eb0a9ebbd617874270541b6be01/spec/lib/methods_spec.rb#L6-L8)

Many users want schema path to be changeable.